zerver/fixtures/slack_fixtures/user_data.json

```diff
@@ -0,0 +1,199 @@
+{
+    "ok": true,
+    "members": [
+       {"id": "U6006P1CN",
+        "team_id": "T5YFFM2QY",
+        "name": "k._",
+        "deleted": false,
+        "color": "9f69e7",
+        "real_name": "Josef K.",
+        "tz": "Europe\/Amsterdam",
+        "tz_label": "Central European Time",
+        "tz_offset": 3600,
+        "profile": {
+            "title": "",
+            "phone": "",
+            "skype": "",
```

Comment:
Note: this is a newly added field, not present in the houstondavis dataset. Maybe including the phone also. If so, then they are not preserved when being converted to Zulip format.

Reply:
My position is that this should be documented even though it is minor. Better exhaustive than sorry.